### ISSUE TITLE
(Documentation) Designing GUIs with a GUI

### ISSUE THREAD

**Issue by ShalokShalom** [pauldotknopf/net-core-qml#225]
Just a heads up, Qt just released the Qt Design Studio 1.6, which is available as a gratis community edition since 1.3.

I was not very well aware about this, because they are hiding it quite good. Its not open source, but its free and you can create UIs via drag-n-drop, so without coding and it spits out QML at your liking. Quite convenient, especially when working with designers, who can do their own designs:

https://download.qt.io/official_releases/qtdesignstudio/

**Comment by vadi2** [comment]
It seems to really struggle with QML for qmlnet - for me the form editor doesn't display anything and the qml emulation layer crashes.

**Comment by ShalokShalom** [comment]
Yeah, I switched now to [Godot](https://medium.com/swlh/what-makes-godot-engine-great-for-advance-gui-applications-b1cfb941df3b) anyway 🙄

**Comment by ghost** [comment]
> 

@vadi2, I had to comment out the import statements for my .Net objects, and then the tool was happy with the syntax of my .qml file, and showed the form design and the widgets in the toolbox. Afterward, you can un-comment your imports for compile/execution. That's what I had to do, anyway.

**Comment by vadi2** [comment]
Yeah, I do the same here.
